src/Take.Blip.Builder/Actions/ProcessHttp/ProcessHttpAction.cs

```diff
@@ -81,10 +81,26 @@ public override async Task ExecuteAsync(IContext context, ProcessHttpSettings se
             catch (Exception ex)
             {
                 _logger.Warning(ex, $"An exception occurred while processing HTTP action");
+                if (ex is System.Threading.Tasks.TaskCanceledException)
+                {
+                    CheckActionTimeout(context);
+                }
             }
 
         }
 
+        private void CheckActionTimeout(IContext context)
```

Comment:
Isn't a good name, you are not checking anything
```suggestion
        private void PushTimeoutWarning(IContext context)
```

(You will have to change the calling on :86)